Sources/PrimerSDK/Classes/User Interface/Checkout/VaultCheckoutViewController.swift

```diff
@@ -103,7 +103,18 @@ extension VaultCheckoutViewController: UITableViewDelegate, UITableViewDataSourc
 
         switch option.type {
         case .applePay:
-            router.show(.applePay)
+            let router: RouterDelegate = DependencyContainer.resolve()
+            let rvc = router.root
+            rvc?.dismiss(animated: true, completion: {
+                DispatchQueue.main.async { [weak self] in
+                    let appleViewModel: ApplePayViewModelProtocol = DependencyContainer.resolve()
+                    appleViewModel.payWithApple { (err) in
+                        if let err = err {
+                            print(err)
```

Comment:
[Feedback]
Doesn't look like a good error management to just print an error in the logs 😅

Reply:
Same case as above comment, but from the universal checkout entry point. 

https://github.com/primer-io/primer-sdk-ios/pull/72#discussion_r639692573

Reply:
Removed it completely. That was only for me.